ZeachCobbler.user.js

```diff
@@ -1885,7 +1893,6 @@ jQuery("#connecting").after('<canvas id="canvas" width="800" height="600"></canv
         Pa();
     }
     function Na(a) {
-        H *= Math.pow(0.9, a.wheelDelta / -120 || (a.detail || 0));
```

Comment:
Rather than just deleting the line, can you comment it out and write /*new*/ ? Until I get around to making the code base dynamically injected, these primitive signposts are how I am able to take my changes and apply them against an updated codebase.